examples/components/data_table/selectable_usage.html

```diff
@@ -17,7 +17,7 @@
   </tr>
   </thead>
   <tbody>
-  <tr *ngFor="#material of materials"  md-data-table-selectable-row [selectable-value]="material.id">
+  <tr *ngFor="#material of materials"  md-data-table-selectable-row [selectable-value]="material.id" [is-active]="material.id == 2">
```

Comment:
I don't mind it, just comparing the id to `2` isn't a really clearly useful example though.